google/cloud/security/notifier/pipelines/slack_webhook_pipeline.py

```diff
@@ -0,0 +1,95 @@
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Slack webhook pipeline to perform notifications"""
+import json
+import requests
+
+# TODO: Investigate improving so we can avoid the pylint disable.
+# pylint: disable=line-too-long
+from google.cloud.security.common.util import log_util
+from google.cloud.security.notifier.pipelines import base_notification_pipeline as bnp
+# pylint: enable=line-too-long
+
+
+LOGGER = log_util.get_logger(__name__)
+
+TEMP_DIR = '/tmp'
+VIOLATIONS_JSON_FMT = 'violations.{}.{}.{}.json'
+OUTPUT_TIMESTAMP_FMT = '%Y%m%dT%H%M%SZ'
+
+
+class SlackWebhookPipeline(bnp.BaseNotificationPipeline):
+    """Slack webhook pipeline to perform notifications"""
+
+    def _compose(self, **kwargs):
+        """Composes the slack webhook content
+
+        Args:
+            **kwargs: Arbitrary keyword arguments.
+
+        Returns:
+            webhook_payload: a string formatted violation
+        """
+
+        def slack_output_dump(data, indent=0):
```

Comment:
Another nit: method names usually start with a verb.  So, in this case, the method name would be more awesome as:
```_dump_slack_output()```